monasca-agent-collector/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM monasca/agent-base:master-20170901-210328
+ARG MON_AGENT_BASE_VERSION
```

Comment:
This won't have any effect, `ARG` needs to be before `FROM` and then needs to actually be used.

Alternatively, we can just update the `FROM` and not have an `ARG` at all, which IMO is a bit simpler and still works fine since we don't have a python3 agent to worry about.

Reply:
actually I just merged https://github.com/monasca/monasca-docker/pull/219 which will cover this as well